src/factory/PimlicoFactory.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import "./FactoryPaymaster.sol";
+import "./FactoryOracle.sol";
+
+
+contract PimlicoFactory is FactoryPaymaster, FactoryOracle {
```

Comment:
can we call it something other than Pimlico? it'd be nice to not include the name Pimlico in anything here